### DISCUSSION TITLE
lost in servers

### DISCUSSION THREAD

**Discussion by jprykiel** [mastodon/mastodon#23503] [Q&A]
Dear all.
Yesterday I created my first Mastodon account. Being French, and French being my native language, I chose the default French server which was aquilenet.fr.
I then started experimenting, and followed only one person for a start. I then found out that his subscriptions/subscribers list, among which I was looking for someone else, were not available because not on the same server. So I deleted my account and create a new one with the same credentials, using the main server, Mastodon.social. On this one, I couldn't find the above-mentioned person, as his name showed no results.
Can you help?
Thanks,
JPR

**Comment by ledgeri** [comment]
@jprykiel  You have **an** account on one server, but you can see the whole network*! So if you know what is the individual's ID, you can find it from any account on any server*! But here is the reason for the stars (*): because anyone can run a server, with any server-specific rule-set, some servers limit others (the mentioned rulesets are disagree on postable content, or content markings, or just the usual spam). So you might can not follow someone, but probalby you will be able to!

As they mention in the FAQ -as far i remember- This working as emails: you can have one or more email accounts, and addresses, but depending on your own provider, you might can not send to, or recive from given addresses, but mostly everithing is compatible with everithing! Going with the examle: if you want to find that person, you have to find that person with the full ID/LINK, of that person, because you can not send an email just to "person", you need "person@provider.xy"... On mastodon you have to give the correct "@person@mastodon-protokol-server.xy", because the "@person" is not enough (basicly it is only searching within your own server, but you get the point!
